Sources/MapboxNavigation/OrnamentsController.swift

```diff
@@ -284,7 +333,11 @@ extension NavigationMapView {
                     }
                     
                     var hideWayName = true
-                    if smallestLabelDistance < 5 {
+                    if latestFeature != similarFeature {
+                        if self.navigationView.wayNameView.setupWith(roadFeature: similarFeature!, using: self.navigationMapView.mapView.style) {
```

Comment:
Please do additional check to verify that `similarFeature` and `latestFeature` do not contain `nil` to prevent force unwrapping.

Reply:
Thanks for the suggestion, I'll add the check.